app/src/main/java/com/teamagam/gimelgimel/app/control/receivers/NewLocationBroadcastReceiver.java

```diff
@@ -23,7 +23,8 @@ public void onReceive(Context context, Intent intent) {
             LocationSample loc = intent.getParcelableExtra(
                     LocationFetcher.KEY_NEW_LOCATION_SAMPLE);
             Log.v(LOG_TAG, "New Location: " + loc.toString());
-            GGMessagingUtils.sendUserLocationMessageAsync(loc);
+            GGMessageSender messageSender = new GGMessageSender(context);
```

Comment:
Make GGMessageSender a dependency for this class.
Meaning, it gets an instance at the constructor.
Use can use a static factory method to instantiate one from context. !fix